sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/Generated/Models/CreateDataFlowDebugSessionRequest.cs

```diff
@@ -80,7 +80,7 @@ public CreateDataFlowDebugSessionRequest(string computeType = default(string), i
         /// debug session.
         /// </summary>
         [JsonProperty(PropertyName = "integrationRuntime")]
-        public IntegrationRuntimeResource IntegrationRuntime { get; set; }
+        public IntegrationRuntimeDebugResource IntegrationRuntime { get; set; }
```

Comment:
I assume this breaking change is intentional and covered by the version bump to 4.2.0?

Reply:
@AlexGhiondea This API hasn't been released yet, we are releasing it now.